app/models/concerns/atmosphere/user_ext.rb

```diff
@@ -45,5 +46,40 @@ def vph_find_or_create(auth)
 
       user
     end
+
+    def jwt_find_or_create(auth)
+
```

Comment:
Please compact a little bit this method by removing extra empty lines. What is more please re-think if all debug messages are needed in this method.